src/components/menu/Menu.js

```diff
@@ -0,0 +1,105 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import { getDishById } from './helpers/menuHelper';
+import _omit from 'lodash/omit';
+
+//Constants
+import { INITIAL_STATE, CHECKOUT } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = INITIAL_STATE;
+
+  resetCartItems = () => {
+    this.setState({
+      cartItems: [],
+      quantities: {},
+    });
+  };
+
+  handleEvents = (dishId = CHECKOUT, changeInQuantity = 1) => {
```

Comment:
instead of handle events we can create separate functions for checkout and cart change that will be more meaningful and easy to understand